forest/jasmine/traj2stats.py

```diff
@@ -1,14 +1,67 @@
+"""Module used to impute missing data, by combining functions defined in other
+modules and calculate summary statistics of imputed trajectories.
+"""
+
 import os
-import sys
 import pickle
+import sys
+import time
+from typing import Tuple
+from functools import partial
+
 import numpy as np
 import pandas as pd
-from ..poplar.legacy.common_funcs import (stamp2datetime, datetime2stamp,
-                                          read_data, write_all_summaries)
-from .data2mobmat import (great_circle_dist, pairwise_great_circle_dist,
-                          GPS2MobMat, InferMobMat)
-from .mobmat2traj import num_sig_places, locate_home, ImputeGPS, Imp2traj
-from .sogp_gps import BV_select
+import pyproj
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+from shapely.ops import transform
+
+from forest.poplar.legacy.common_funcs import (datetime2stamp, read_data,
+                                               stamp2datetime,
+                                               write_all_summaries)
+from forest.jasmine.data2mobmat import (GPS2MobMat, InferMobMat,
+                                        great_circle_dist,
+                                        pairwise_great_circle_dist)
+from forest.jasmine.mobmat2traj import (Imp2traj, ImputeGPS, locate_home,
+                                        num_sig_places)
+from forest.jasmine.sogp_gps import BV_select
+
+
+def transform_point_to_circle(coords: Tuple[float, float], radius: int
```

Comment:
If `coords` has to be unpacked anyway then it makes sense to pass `lat` and `lon` directly instead.